xknxproject/models/knxproject.py

```diff
@@ -77,6 +77,18 @@ class GroupAddress(TypedDict):
     dpt: DPTType | None
     communication_object_ids: list[str]
     description: str
+    comment: str
+
+
+class GroupRange(TypedDict):
+    """GroupRange typed dict."""
+
+    name: str
+    address_start: str
+    address_end: str
+    comment: str
+    children: dict[str, GroupRange]
```

Comment:
Personally I'd prefer somthing like `sub_groups` or `middle_groups`. We can only have one sub-element anyway - and we know beforehand since the GA-style is known.

Reply:
My changes will propose 'group_ranges' as attribute for the main group ranges - like discussed on discord